projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -70,8 +88,20 @@ export class TabParagraphContainerComponent
     )
   );
 
-  select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+  select(tabNum: number, event?: Event): void {
+    this.breakpointService
```

Comment:
Since `breakpointService` is optional, it can be null and we should treat it as possibly-null, i.e. by using optional chaining (I mean `?.`): `this.breakpointService?.isDown(...)....`

The same will apply to `winRef`.